x-pack/plugins/observability/common/custom_threshold_rule/types.ts

```diff
@@ -166,7 +166,6 @@ export enum MetricsExplorerChartType {
 export enum InfraRuleType {
```

Comment:
I don't know why we have this type in observability plugin, we should clean this up. (We should check it at https://github.com/elastic/kibana/issues/159340)